### ISSUE TITLE
Dashboard ids hashing leads to unexpected UX

### ISSUE THREAD

**Issue by berekuk** [QURIresearch/metaforecast#61]
If I go to https://metaforecast.org/dashboards and create a dashboard with example ids but a modified title, I'll get someone else's dashboard (e.g., the default one), because INSERT in `pgInsertIntoDashboard` fails quietly and dashboard id is built from question ids only.

Is this a feature or a bug? Seems more like a bug to me, but I'm not entirely sure.

Maybe we should switch to the random uuids/[cuid](https://github.com/ericelliott/cuid)s instead of relying on hashing?

**Comment by NunoSempere** [comment]
Yeah, this was a badly thought out feature, is fine to change. I'd have a weak preference for SHA-hashing the whole object, rather than just the array of ids
